### ISSUE TITLE
PR preview only works in branches of this repo, not in forks

### ISSUE THREAD

**Issue by hamoid** [prachtsaal/prachtsaal.berlin#79]
See https://github.com/rossjrw/pr-preview-action/discussions/95

I guess we need a Contribute section, where it explains that one should close the repo, ask to be added as a contributor, and maybe point out  at this issue. Or maybe we can tweak the PR preview action not to run unless the repo is this one?

**Comment by mingness** [comment]
Could migrate pr previews to netlify as well.

**Comment by hamoid** [comment]
Thank you for the offer :-)

Personally, I preview on my computer. I don't think others have used it. I would find it easier if I could just push changes to this repo directly, and people send me requests when something needs to be updated. Currently (side topic) other people need to approve changes, and not many Prachtsaalians have the interest / time / knowledge to understand if changes are good or not. It can be a bit frustrating to have to wait and see changes live involving others who are busy to blindly click accept. I think the current approach is a bit overburocratic for the actual size of the contributing team and the not-so-essential material we publish once in a while.

What do you think?

**Comment by mingness** [comment]
True. The point of the approval step is to have another pair of eyes on the change, and often that doesn't happen. So, we could remove the need for review.

I still think having a preview is good, when doing updates that involve other people, but as I'm not in the middle of edits, I won't impose my thinking on what is practically useful at the moment where you are.

**Comment by hamoid** [comment]
For the previews, as long as we use branches instead of forks it's all good. We could document the steps: make sure your user is added to the Prachtsaal GitHub, then create a new branch, do your thing, push the branch, enjoy the preview :-)
